src/menu.js

```diff
@@ -0,0 +1,71 @@
+class Menu {
+  constructor(){
+    this.items = {
+      fries: {
+        name: 'Fries', price: 1.99, id: 1
```

Comment:
If your table serialises the IDs, but you provide hard-coded IDs in the constructor - do you not run into issues with IDs already existing?